lib/Runtime/Base/JnDirectFields.h

```diff
@@ -416,7 +416,7 @@ ENTRY(valueOf)
 ENTRY(values)
 ENTRY(WeakMap)
 ENTRY(WeakSet)
-ENTRY(WinRTError)
+ENTRY(WinRTError) // remove this upon next bytecode regen
```

Comment:
I didn't actually delete this because that would have required a bytecode regen and I thought I'd leave that until I had another reason for one